src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -245,27 +279,71 @@ public int getNextOpenSensor() {
   public String senseColor() {
     if (currentColorCycle % cycleWait == 0) {
       currentColorCycle = 0;
-      Color[] colors = colorSensors.getColors();
-      SmartDashboard.putNumber("Front Sense B", colors[2].blue);
-      SmartDashboard.putNumber("Front Sense R", colors[2].red);
+      colors = colorSensors.getColors();
+
+      double magnitude = colors[2].red + colors[2].green + colors[2].blue;
+      double redRatio = colors[2].red / magnitude;
+      double blueRatio = colors[2].blue / magnitude;
```

Comment:
This turns out to be redundant. ColorSensorV3 already scales this way. So the magnitude here will always be 1